x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -151,11 +151,26 @@ aggField
     ;
 
 qualifiedName
+    // TODO: Test all kind of valid/invalid qualifier strings and make sure they make sense. Same for patterns below.
+    // TODO: Account for qualifiers in parameters.
```

Comment:
Are they intentionally left o later? 
If so, it is worth opening issue or list item in epic to remember to resolve/remote those?